update-submodule/action.yml

```diff
@@ -90,6 +94,7 @@ runs:
         git push --force origin "${{ inputs.feature_branch }}"
 
     - name: Create a pull request against the main branch
+      if: create_pr == "true"
```

Comment:
Looks like should be `inputs.create_pr`.

Reply:
That's what happens when we don't test our code.

Reply:
fixed